src/app/pages/send/send-crypto-asset-form/components/recipient-type-dropdown/components/recipient-dropdown-item.tsx

```diff
@@ -17,23 +19,23 @@ export function RecipientDropdownItem({
 }: RecipientDropdownItemProps) {
   return (
     <styled.button
+      display="flex"
+      pos="relative"
+      type="button"
+      color="accent.text-primary"
       _hover={{
         bg: isVisible ? 'accent.component-background-hover' : 'accent.background-primary',
         borderRadius: 'xs',
-        color: 'accent.text-primary',
+        color: 'accent.action-primary-hover',
       }}
       alignItems="center"
       data-testid={`recipient-select-field-${testLabels[index]}`}
-      display="flex"
-      height="30px"
-      minWidth="110px"
       onClick={() => onSelectItem(index)}
+      zIndex={20}
       pl={isVisible ? 'space.02' : 'unset'}
-      textStyle="label.02"
-      zIndex={999}
     >
       <HStack gap="space.01">
-        <styled.span textStyle="label.02">{labels[index]}</styled.span>
+        <styled.span textStyle="label.03">{labels[index]}</styled.span>
```

Comment:
I think the below 
```
{isVisible ? <></> : <ChevronDownIcon variant="small" />}
```

can just be:
{isVisible && <ChevronDownIcon variant="small" />}